client/query/query.go

```diff
@@ -144,16 +145,20 @@ func GetAllLedgerChannels(store store.Store, consensusAppDefinition types.Addres
 	return toReturn, nil
 }
 
-func GetPaymentChannelsByLedger(ledgerId types.Destination, store store.Store, vm *payments.VoucherManager) ([]PaymentChannelInfo, error) {
+func GetPaymentChannelsByLedger(ledgerId types.Destination, s store.Store, vm *payments.VoucherManager) ([]PaymentChannelInfo, error) {
 	// If a ledger channel is actively funding payment channels it must be in the form of a consensus channel
-	con, err := store.GetConsensusChannelById(ledgerId)
+	con, err := s.GetConsensusChannelById(ledgerId)
+	// If the ledger channel is not a consensus channel we know that there are no payment channels funded by it
+	if errors.Is(err, store.ErrNoSuchChannel) {
+		return []PaymentChannelInfo{}, nil
```

Comment:
Not sure what the correct behaviour ought to be here. If I query the payment channels for some made up ledger id, should i get an empty list or an error?

Reply:
There's two possibilities here:
1. Querying for a made up ledger id.
2. Querying for a ledger channel that is no longer a consensus channel(Ie: it's being defunded)

I wanted to avoid throwing an error in 2.